Bio/motifs/__init__.py

```diff
@@ -24,16 +24,17 @@ def create(instances, alphabet=None):
 
 
 def parse(handle, format):
+    
```

Comment:
That's the wrong place for the blank line. It should be inside the docstring,

```python
def parse(handle, format):
    """Parse an output file from a motif finding program.

    Currently supported formats (case is ignored):
    ...
```

Reply:
This is why you still have the D205 error, and a new D201 error. Also it seems this is not really a blank line, but has some spaces - triggering the W293 warning as well:

```
Bio/motifs/__init__.py:26:1: D205 1 blank line required between summary line and description
Bio/motifs/__init__.py:26:1: D201 No blank lines allowed before function docstring
Bio/motifs/__init__.py:27:1: W293 blank line contains whitespace
```